lib/robots/dor_repo/accession/end_accession.rb

```diff
@@ -22,6 +22,14 @@ def current_version
           @current_version ||= object_client.version.current
         end
 
+        # we will pass on any workflow context to the next version when starting OCR or speechToText
+        # so that user selections like language will be available to the OCR/speechToText workflows,
+        # but we will remove the context values that actually starts OCR/speechToText when doing this,
+        # or else we will end up in an infinite loop, where the workflow gets triggered over and over again
+        def workflow_context_for_next_version
+          workflow.context.except('runOCR', 'runSpeechToText')
```

Comment:
Is it dangerous to pass on all context if we start using context more broadly? Should you just pass on the specific context variables that you know you want?

Reply:
could consider this, advantage is that we limit what is passed on, disadvantage is that we'd need to remember to add them here if we add things in pre-assembly which add to the context

i'm a bit ambivalent since this bit of code is only ever triggered if the user specifically asks for OCR or speech to text, so is for a small subset of objects (admittedly the only ones that i am aware of taking advantage of this context feature)